helm/dashboards/charts/private_dashboards_al/dashboards/shared/private/capa-aggregated-error-logs.json

```diff
@@ -106,7 +106,7 @@
           "type": "prometheus",
           "uid": "${prometheus_datasource}"
         },
-        "definition": "label_values(up{app=\"kubernetes\"},cluster_id)",
+        "definition": "label_values(up{job=\"kubernetes\"},cluster_id)",
```

Comment:
For `app=kubernetes`, job is equal to `apiserver`, not `kubernetes`.
You can rely on `service=kubernetes`